site/app/libraries/database/DatabaseQueries.php

```diff
@@ -1119,11 +1119,11 @@ public function getGradeablesPastAndSection() {
      */
     public function getCountUsersRotatingSections() {
         $this->course_db->query("
-SELECT rotating_section, count(*) as count
-FROM users
-WHERE registration_section IS NOT NULL
-GROUP BY rotating_section
-ORDER BY rotating_section");
+            SELECT rotating_section, count(*) as count
```

Comment:
Why change this indenting?  
This is now different than on queries in this file.  I don't see functional changes on these lines.

Reply:
(And the others below as well)